src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -4584,5 +4584,133 @@ static T F2<T>(T x2)
             };
             AssertEx.Equal(expectedAnnotations, actualAnnotations);
         }
+
+        [Fact]
+        public void AutoPropInitializer_01()
+        {
+            var source = @"
+class C
+{
+    public string Prop { get; set; } = ""a"";
+    public C()
+    {
+        Prop.ToString();
+    }
+
+    public C(string s) : this()
+    {
+        Prop.ToString();
+    }
+}
+";
+            var comp = CreateCompilation(source, options: WithNonNullTypesTrue());
+            comp.VerifyDiagnostics();
+
+            var tree = comp.SyntaxTrees.Single();
+            var model = comp.GetSemanticModel(tree);
+
+            var memberAccesses = tree.GetRoot().DescendantNodes().OfType<MemberAccessExpressionSyntax>().ToArray();
+            Assert.Equal(2, memberAccesses.Length);
+
+            var receiver = memberAccesses[0].Expression;
+            var info = model.GetTypeInfo(receiver);
+            Assert.Equal(PublicNullableAnnotation.NotAnnotated, info.Type.NullableAnnotation);
+            Assert.Equal(PublicNullableFlowState.NotNull, info.Nullability.FlowState);
```

Comment:
>Nullability.FlowState [](start = 63, length = 21)

Please verify the nullable state on the type symbol. We really need to deprecate these apis.

Reply:
> Please verify the nullable state on the type symbol.

Is that the same thing as checking `info.Type.NullableAnnotation` which is done on the previous line?

> We really need to deprecate these apis.

Is this in reference to the 'Nullability' property entirely?

Reply:
Ah, yes, I misread that. And yes, I mean we should really deprecate the `Nullability` property.

---
In reply to: [470190746](https://github.com/dotnet/roslyn/pull/46441#discussion_r470190746) [](ancestors = 470190746)